DeveloperConsole.cs

```diff
@@ -7,8 +7,9 @@
 using UnityEngine;
 using UnityEngine.UI;
 using UnityEngine.InputSystem;
+using UnityEngine.InputSystem.LowLevel;
```

Comment:
What is this used for?

Reply:
Okay, I thought it was added in on your side. I'll remove that